sstables/sstables.cc

```diff
@@ -1495,7 +1495,7 @@ void sstable::maybe_rebuild_filter_from_index(uint64_t num_partitions) {
     bloom_filter_builder bfb_consumer(optimal_filter);
     index_consume_entry_context<bloom_filter_builder> consumer_ctx(
             *this, sem.make_tracking_only_permit(_schema, "rebuild_filter_from_index", db::no_timeout, {}), bfb_consumer, trust_promoted_index::no,
-            make_file_input_stream(index_file, 0, index_file_size, {.buffer_size = sstable_buffer_size}), 0, index_file_size,
+            make_file_input_stream(index_file, 0, index_file_size, file_input_stream_options{.buffer_size = sstable_buffer_size}), 0, index_file_size,
```

Comment:
Looks like an unrelated change, why is it needed as part of this patch?

Reply:
good catch, thanks. removed.